### PR TITLE
feat : added home-page

### PR DESCRIPTION
**Pull request by sach10-create**

### CODE REVIEW

File: css/home.css [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+/* .ecom-header .social-icon-container .icon-badge-container {
+  display: flex;
+}
+
+.ecom-header .social-icon-container .icon-badge {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+} */
```
Comment (iRohitGaur) [inline css/home.css]: remove commented code

File: css/home.css [hunk 2]
```diff
@@ -0,0 +1,270 @@
+@import url("https://fonts.googleapis.com/css2?family=Secular+One&display=swap");
+/* header */
+
+.ecom-header {
+  color: var(--white);
+  background: #29283d;
```
Comment (iRohitGaur) [inline css/home.css]: use variables

File: css/home.css [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+.ecom-home-container .logo-container p {
+  left: 11.5rem;
+  top: 6.5rem;
+  color: var(--white);
+}
+/* Home Page */
+.ecom-home-container {
+  display: grid;
+  grid-template-columns: repeat(8, 1fr);
+  /* grid-auto-rows: minmax(200px, auto); */
```
Comment (iRohitGaur) [inline css/home.css]: remove commented code

File: css/home.css [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+  grid-template-areas:
+    "header header header header header header header header"
+    "offer offer offer offer offer offer offer offer"
+    ". main main main main main main ."
+    ". section section section section section section ."
+    ". section2 section2 section2 section2 section2 section2 ."
+    ". section3 section3 section3 section3 section3 section3 ."
+    ". section4 section4 section4 section4 section4 section4 ."
+    ". section5 section5 section5 section5 section5 section5 ."
+    "footer footer footer footer footer footer footer footer";
```
Comment (iRohitGaur) [inline css/home.css:81]: this can be achieved in a shorter way

File: css/home.css [hunk 5]
```diff
@@ -0,0 +1,270 @@
...
+}
+
+/* Section */
+.ecom-section-container .top-picks-heading {
+  font-size: 5rem;
+  text-align: center;
+  padding: 1rem;
+  margin: 6rem 0;
+  width: 100%;
+  border-bottom: 5px solid #cd122d;
```
Comment (iRohitGaur) [inline css/home.css]: use variables

File: css/home.css [hunk 6]
```diff
@@ -0,0 +1,270 @@
...
+  font-family: "Secular One", sans-serif;
+}
+
+.product-category-container {
+  display: flex;
+  gap: 2rem;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+  border-bottom: 5px solid #cd122d;
```
Comment (iRohitGaur) [inline css/home.css]: use variables

File: css/home.css [hunk 7]
```diff
@@ -0,0 +1,270 @@
...
+  left: 50%;
+  transform: translate(-50%);
+  padding: 1rem;
+  background-color: #d4d4d8;
+  width: 85%;
+  border-radius: 2rem;
+  text-align: center;
+  opacity: 0.9;
+  /* background: transparent; */
+  /* margin: 10px; */
```
Comment (iRohitGaur) [inline css/home.css]: remove commented code

File: css/home.css [hunk 8]
```diff
@@ -0,0 +1,270 @@
...
+}
+
+.ecom-section4-container > *:hover {
+  cursor: pointer;
+}
+
+.ecom-section5-container > *:hover {
+  cursor: pointer;
+}
+
```
Comment (iRohitGaur) [inline css/home.css:245]: same properties in the classes, you can combine them using comma

**Review by iRohitGaur — APPROVED** [review]
merge with changes

**Review by sonishreyas — APPROVED** [review]
Nothing much to add in the comments above. Rest looks good

**Pull request merged**